src/routes/shop/body/+page.svelte

```diff
@@ -114,7 +114,7 @@
     <div class="technical-specs">
       <h1>Technical Specs</h1>
       <div class="chart">
-        <Grid columns={3}>
+        <Grid columns={3} wrapMode="single"> 
```

Comment:
why were these all changed?

Reply:
Explained in my longer comment, but basically the fix I did to the Grid component changed the behavior everywhere `columns` is an odd number with `wrapMode="half"` or `wrapMode` left unset. Adding `wrapMode="single"` essentially forces using a single column, which I believe was already happening on mobile due to the bug.